youths/models.py

```diff
@@ -0,0 +1,61 @@
+import reversion
+from django import forms
+from django.conf import settings
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.forms import SelectMultiple
+
+from profiles.models import Profile
+
+
+class ChoiceArrayField(ArrayField):
+    """
+    A field that allows us to store an array of choices.
+    Uses Django's Postgres ArrayField and a MultipleChoiceField for its formfield.
+    Adapted from https://gist.github.com/danni/f55c4ce19598b2b345ef
+    """
+
+    def formfield(self, **kwargs):
+        defaults = {
+            "form_class": forms.MultipleChoiceField,
+            "choices": self.base_field.choices,
+            "widget": SelectMultiple,
+        }
+        defaults.update(kwargs)
+        return super(ArrayField, self).formfield(**defaults)
+
+
+@reversion.register()
+class YouthProfile(models.Model):
+    # Required info
+    profile = models.OneToOneField(Profile, on_delete=models.CASCADE)
+    ssn = models.CharField(max_length=11)  # TODO ssn validation?
+    school_name = models.CharField(max_length=128)
+    school_class = models.CharField(max_length=10)
+    expiration = models.DateField(null=True, blank=True)
+
+    # Optional info
+    preferred_language = models.CharField(
```

Comment:
why not just use the [`language` field of the `Profile` model](https://github.com/City-of-Helsinki/open-city-profile/blob/master/profiles/models.py#L44)?

Reply:
maybe make the `YouthProfileAdmin` as inline of the `ProfileAdmin`? or can we have youth profiles without regular profiles?

Reply:
There were only fi, sv, and en in the `Profile` language choices, whereas this could have some rarer languages (some youth houses offer services in Somalian, Arabic, and maybe some other language as well). Not sure about this one

Reply:
yeah, okay, let's keep this field. it probably should use a different tuple of options then

Reply:
About the inline admin widgets, good point. `YouthProfile` can't exist withouth `Profile`.

Reply:
Profiles were inlined into `UserAdmin` and as far as I can tell, nested inlines aren't possible/easy without installing additional dependencies. I opted to expose `Profile`s as their own admin page, with `YouthProfile`s inlined.